README.md

```diff
@@ -1,10 +1,22 @@
-# accessibility-statement-frontend
+# Accessibility Statement Frontend
 
 Accessibility Statement Frontend is a service for providing for accessibility statements across HMRC.
 
+## Prerequisites
+
+Adding an accessibility statement involves creating a structured text file known as
+a [YAML](https://en.wikipedia.org/wiki/YAML) (YAML Ain't Markup Language) file and 
+adding to this repository.
+
+Creating such a file does not require any coding skills and can be done in a simple text editor, online 
+ YAML editor or even directly within Github. However, if you are a developer you may 
+prefer to use your preferred IDE and git to upload the YAML file to the repository.
```

Comment:
"git", or "Git"? I genuinely don't know.

Reply:
I'm feeling you'd prefer Git?

Reply:
:)